integration-test/src/nativeMain/kotlin/Dummy.kt

```diff
@@ -1,2 +1,2 @@
 // This is a workaround for https://github.com/JetBrains/kotlin-native/issues/2617
-object Dummy
+object Dummy
```

Comment:
issue close now, but it didn't make it on the `1.3.70` release. So either in `1.3.71` or `1.4.0`.